src/schema/ecommerce/types/order.ts

```diff
@@ -134,4 +136,10 @@ export const {
   edgeType: OrderEdge,
 } = connectionDefinitions({
   nodeType: OrderType,
+  connectionFields: {
+    totalCount: {
+      type: GraphQLInt,
+      resolve: ({ totalCount }) => totalCount,
```

Comment:
wanna humor me and see if we can remove this line?

Reply:
looks like a very removable line... testing now.